hawezo/js-translation-bundle/0.1/config/packages/dev/js_translation.yaml

```diff
@@ -0,0 +1,5 @@
+js_translation:
+    translation_extract_path: 'assets/js/messages.js'
+    auto_extract: true
+    export_domains: []
+    export_locales: []
```

Comment:
You dont need to duplicate the values from `config/packages/js_translation.yaml`. It would be enough if this file only contained `auto_extract`